app/lambda/datastore.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, databaseName, tableName, partitionKeyName, sortKeyName=None,
         self._indexPartitionKeyName = indexPartitionKeyName
         self._indexSortKeyName = indexSortKeyName
     
-    def save(self, info, awsRegion=None):
+    def save(self, info, awsRegion='us-west-2'):
```

Comment:
You should not add in the aws region as a parameter to all APIs. The region is going to be the same for the entire database and hence should be a part of init.